octoprint_mrbeam/templates/conversion_dialog.jinja2

```diff
@@ -287,6 +287,29 @@
 								</div>-->
 							</div>
 
+                            <div class="control-group">
+                                <div data-toggle="tooltip" data-placement="right"
+                                     data-bind="css: showExpertSettings() == true ? 'progressive-control-and-label' : 'engraving-control-and-label'"
+                                     title="<b>{{ _('Indicates the number of times the laser will run over the design to cut it. ') }}</b>
+                                            {{ _('Increase the number of passes if the material does not get cut.') }}<br>
+                                            <ul>
+                                            <li>{{ _('Progressive cut (activate in advanced settings):') }}
+                                            {{ _('Slows down the laser head progressively in every cut.
+                                            Depending on the design, this might save time by making it possible to reduce passes, and can also reduce burn marks.') }}
+                                            </li></ul>">
+                                    <label class="control-label">{{ _('Passes') }}</label>
+                                    <div class="controls">
+                                        <input data-bind="value: engravingPasses, event: {change: flag_customized_material}" class="passes_input engrave_passes_input" type="number" />&nbsp;&times;&nbsp;&nbsp;
+                                        <span style="display: inline-block">
+                                            <label data-bind="visible: showExpertSettings">
+                                                <input data-bind="checked: engravingProgressive, event: {change: flag_customized_material}" type="checkbox" class="param_eng_progressive"/>
+                                                {{ _('progressive') }}
+                                            </label>
+                                        </span>
+                                    </div>
+                                </div>
+                            </div>
+
```

Comment:
I think multi-pass engraving would be best suited as a part of the advanced settings as well.

- Change the text to say "engrave" instead of "cut"
- The progressive engraving is probably a bad idea, because it makes it difficult to foresee how deep the engraving gets (progressive cut makes 1 quick pass, then increasingly slow passes)